src/main/webapp/userdir.html

```diff
@@ -22,8 +23,8 @@ <h3>a recommendation system for students, by students!</h3>
     </div>
     <div class="content">
       <h1>User Directory</h1>
-      <div class="user-dir">
-        <div class="user-card">
+      <div id="user-dir">
+        <!-- <div class="user-card">
```

Comment:
As we aren't using this section, I think we can remove it rather than just have it commented out.